### ISSUE TITLE
feat: update todometer

### ISSUE THREAD

**Issue by DirtyF** [cassidoo/todometer#81]
Thank you for this nice little piece of software 🙏 

It would be _nice_ to be able to update Todometer right from the menu as Atom does for instance. 🆕 📦 

Maybe with https://github.com/electron/update-electron-app? ✨

**Comment by cassidoo** [comment]
Yes, I'd love to do this! I think I have to figure out how to make it a signed app first, but definitely want to implement this. Thanks for making an issue.

**Comment by DirtyF** [comment]
Good luck with that! https://www.electronjs.org/docs/tutorial/code-signing
